openpype/hosts/blender/hooks/pre_set_workfile_publish_time.py

```diff
@@ -0,0 +1,20 @@
+from pathlib import Path
+
+from openpype.lib import PreLaunchHook
+from openpype.hosts.blender import utility_scripts
+
+
+class PreSetWorkfilePublishTime(PreLaunchHook):
+
+    def execute(self):
+        if self.data.get("set_workfile_publish_time"):
```

Comment:
I think detecting if `pre_copy_last_published_workfile` has been triggered could be done using one of the value already set by the hook, like `"source_filepath"` instead of adding a new dedicated bool.